src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -206,7 +206,13 @@ class ActiveFocusInvestigation extends React.Component<
             const thePlans = plansArray.map((item: Plan) => {
               let thisItem = extractPlan(item);
               // transform values of this properties if they are null
-              const propertiesToTransform = ['village', 'canton', 'district', 'province'];
+              const propertiesToTransform = [
+                'village',
+                'canton',
+                'district',
+                'province',
+                'focusArea',
+              ];
```

Comment:
These look like magic strings which should be moved to `constants` please

Reply:
@cKellyDesign This actually contains domain logic, they are not just strings, see [here](https://github.com/onaio/reveal-frontend/blob/24e4230c76784a5d0f4df9181e382f3e7b0009e2/src/helpers/utils.tsx#L334).